build/types.go

```diff
@@ -2,10 +2,11 @@ package build
 
 // Config holds the setup variables required for a build
 type Config struct {
-	OS          string // GOOS
-	Arch        string // GOOS
-	EnableDebug bool
-	Env         map[string]string
+	OS                   string // GOOS
+	Arch                 string // GOOS
+	EnableDebug          bool
+	Env                  map[string]string
+	EnableDynamicLinking bool
```

Comment:
Having thought about this some more, I think the real problem isn't actually dynamic linking, but cgo. The Oracle plugin needs cgo, and static linking is just a result of disabling it:
```suggestion
	EnableCGO bool
```